buildconfig/CMake/QtTargetFunctions.cmake

```diff
@@ -220,7 +232,14 @@ endfunction()
 # install_target_type The type of target that should be installed. See
 # https://cmake.org/cmake/help/latest/command/install.html?highlight=install - install_dir A relative directory to
 # install_prefix
-function(mtd_install_qt_library qt_version target install_target_type install_dir)
+function(mtd_install_qt_library qt_version target install_dir)
+  get_target_property(target_type ${target} TYPE)
+  set(install_target_type "")
+  if(WIN32 AND NOT target_type STREQUAL "MODULE_LIBRARY")
+    # Install only DLLs on Windows
+    set(install_target_type RUNTIME)
+  endif()
```

Comment:
The indentation in this block seems to be too much

Reply:
Lines 236 through to 241, I think should be unindented. When compared to line 235

Reply:
Oh I see. Yes that's intentional because line 236 is the start of the content of the function that's defined on line 235.

Reply:
Ah whoops, I misread this!